zipp 3.11.0

skupr-anaconda opened the pull request:
**Jira ticket:** [PKG-944](https://anaconda.atlassian.net/browse/PKG-944)

**The upstream data:**
Github releases:  https://github.com/jaraco/zipp/releases
[Diff between the latest and previous upstream releases](https://github.com/jaraco/zipp/compare/v3.8.0...v3.11.0)
Changelog: https://github.com/jaraco/zipp/blob/main/CHANGES.rst
Requirements:
 * setup.cfg:  https://github.com/jaraco/zipp/blob/v3.11.0/setup.cfg
 *  pyproject.toml:  https://github.com/jaraco/zipp/blob/v3.11.0/pyproject.toml

**_Actions:_**

- Add upper bound to get around the circular dependency `zipp->setuptools_scm->importlib_metadata->zipp`, see https://github.com/conda-forge/zipp-feedstock/commit/94b9a99af3adb8060fb705cefe27f851028f9b9a

- Update tests: source_files, requires, commands

- Add a description

- Fix doc_url

**Additional info:**

<details>

**Package's statistics**

<details>

 * Priority C | effort: easy | Category: anaconda | subcategory:  | pkg_type: python
 * Outdated platfroms: 'linux-64', 'linux-aarch64', 'linux-s390x', 'linux-ppc64le', 'osx-64', 'osx-arm64', 'win-64'
 * Latest version: 3.11.0
 * Release on PyPi:
    * version: 3.11.0
    * date: 2022-11-25T18:12:51
* [PyPi history](https://pypi.org/project/zipp/#history)
 * Popularity: 
    * 3 months downloads: 2557327.0
    * All time:  21221622 downloads -  zipp  3.11.0  A pathlib-compatible Zipfile object wrapper  

</details>

**Other checks:**

<details>

1. - [x] https://github.com/jaraco/zipp/tree/v3.11.0 exists    
8. - [x] Verify that the `build_number` is correct
9. - [x] has `setuptools`
10. - [x] has `wheel`
11. - [x] `pip` in test
12. - [x] Verify the test section
13. - [x] Verify if the package is `architecture specific`
14. - [x] Verify that private modules are not mentioned in the recipe For example: (_private_module)
15.  - [x] license_file: LICENSE is present
16. - [x] license_family MIT is present
17. - [x] License: MIT
    - [x] License is `spdx` compliant
 * Check if the license identifier has correct name from the SPDX License List

| Identifier         |
|:-------------------|
| MIT                |
| MIT-0              |
| MIT-advertising    |
| MIT-CMU            |
| MIT-enna           |
| MIT-feh            |
| MIT-Modern-Variant |
| MIT-open-group     |
| MITNFA             |
</details>

**Check dependency issues:**

<details>
../aggregate/zipp-feedstock/recipe/meta.yaml
Dependencies: ['pip', 'toml', 'python', 'wheel', 'setuptools_scm >=3.4.1', 'setuptools >=56']


noarch:
- py3.7:  No issues found
- py3.8:  No issues found
- py3.9:  No issues found
- py3.10:  No issues found

linux-64:
- py3.7:  No issues found
- py3.8:  No issues found
- py3.9:  No issues found
- py3.10:  No issues found

linux-ppc64le:
- py3.7:  No issues found
- py3.8:  No issues found
- py3.9:  No issues found
- py3.10:  No issues found

linux-s390x:
- py3.7:  No issues found
- py3.8:  No issues found
- py3.9:  No issues found
- py3.10:  No issues found

osx-64:
- py3.7:  No issues found
- py3.8:  No issues found
- py3.9:  No issues found
- py3.10:  No issues found

osx-arm64:
- py3.8:  No issues found
- py3.9:  No issues found
- py3.10:  No issues found

win-64:
- py3.7:  No issues found
- py3.8:  No issues found
- py3.9:  No issues found
- py3.10:  No issues found

</details>

**Package Build Score: 18**


**Links:**
* [Anaconda Recipes feedstock](https://github.com/AnacondaRecipes/zipp-feedstock)
* [Update branch](https://github.com/AnacondaRecipes/zipp-feedstock/tree/3.11.0)
* [conda-forge recipe](https://github.com/{upstream}/{feedstock_name}-feedstock)
* [PyPI](https://pypi.org/project/zipp)
* [Diff between upstream and feature branch](https://github.com/conda-forge/zipp-feedstock/compare/main...AnacondaRecipes:3.11.0)
* [Diff between origin and feature branch](https://github.com/AnacondaRecipes/zipp-feedstock/compare/master...AnacondaRecipes:3.11.0)
* [The last merged PRs](https://github.com/AnacondaRecipes/zipp-feedstock/pulls?q=is%3Apr+is%3Amerged+sort%3Aupdated-desc+)

</details>

**Updating the recipe:**
If the recipe needs additional modification the update branch can be modified. Note that the PR diffs are not updated with these changes.
```
git clone -b 3.11.0 git@github.com:AnacondaRecipes/zipp-feedstock.git
```

skupr-anaconda committed:
DOR update to 3.11.0

anaconda-pkg-build commented:
<details><summary>Linter check found the following problems:</summary>
The following problems have been found:

ERROR: /tmp/abs_1d5gcpd5e_/clone/recipe/meta.yaml:49: invalid_url: https://zipp.readthedocs.io/en/latest : URL domain redirect zipp.readthedocs.io ->
WARNING: /tmp/abs_1d5gcpd5e_/clone/recipe/meta.yaml:50: missing_doc_source_url: The recipe is missing a doc_source_url
WARNING: /tmp/abs_1d5gcpd5e_/clone/recipe/meta.yaml:50: missing_description: The recipe is missing a description
WARNING: /tmp/abs_1d5gcpd5e_/clone/recipe/meta.yaml:50: missing_license_url: The recipe is missing a license_url
Errors were found

</details>

skupr-anaconda committed:
Release 3.11.0

- Add upper bound to get around circular dependency
    # zipp->setuptools_scm->importlib_metadata->zipp

- Update tests: source_files, requires, commands

- Add a description

- Fix doc_url

[PKG-944]

skupr-anaconda committed:
Remove pytest

anaconda-pkg-build commented:
<details><summary>Linter check found the following problems:</summary>
The following problems have been found:

WARNING: /tmp/abs_148uo2qxnv/clone/recipe/meta.yaml:57: missing_license_url: The recipe is missing a license_url
WARNING: /tmp/abs_148uo2qxnv/clone/recipe/meta.yaml:57: missing_doc_source_url: The recipe is missing a doc_source_url
Warnings were found

</details>

psteyer reviewed (approved):
LGTM

varlackc reviewed (approved):
LGTM

anaconda-pkg-build commented:
<details><summary>Linter check found the following problems:</summary>
The following problems have been found:

WARNING: /tmp/abs_dde6y7e6ic/clone/recipe/meta.yaml:57: missing_license_url: The recipe is missing a license_url
WARNING: /tmp/abs_dde6y7e6ic/clone/recipe/meta.yaml:57: missing_doc_source_url: The recipe is missing a doc_source_url
Warnings were found

</details>

anaconda-pkg-build commented:
<details><summary>Linter check found the following problems:</summary>
The following problems have been found:

WARNING: /tmp/abs_d1zsmvhrqd/clone/recipe/meta.yaml:57: missing_doc_source_url: The recipe is missing a doc_source_url
WARNING: /tmp/abs_d1zsmvhrqd/clone/recipe/meta.yaml:57: missing_license_url: The recipe is missing a license_url
Warnings were found

</details>
